.github/workflows/announce-new-release.yml

```diff
@@ -0,0 +1,46 @@
+name: Send a message on Twitter & Discord
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'The version to announce'
+        type: string
+        required: true
+      description:
+        description: 'A short description of the version to announce'
+        type: string
+        required: true
+
+jobs:
+  tweet:
```

Comment:
can we rename it into "twitter"?

Reply:
Twitter is just the name of the platform.

I don't find it meaningful, because you can do something other than send a tweet.

I can rename it to `send-tweet` or `send-message-on-twitter`.

Reply:
```suggestion
  publish_tweet:
```